gofight.go

```diff
@@ -52,7 +52,9 @@ package gofight
 import (
 	"bytes"
 	"encoding/json"
+	"github.com/gofiber/fiber/v2"
```

Comment:
move github.com to the bottom and add an empty line in the front.

Reply:
Done